magicbyirineu/Modules/CardList/View/Cells/CardCollectionViewCell.swift

```diff
@@ -24,21 +26,23 @@ class CardCollectionViewCell: UICollectionViewCell {
         
         return view
     }()
-    lazy var roundedView: UIView = {
+    let roundedView: UIView = {
```

Comment:
E se isso for uma view componentizada? Para evitar que vc repita esse código na shadowView e nessa view